spec/acceptance/server_spec.rb

```diff
@@ -29,7 +29,7 @@
     # This sleeps because hyperglass can take a long time to start. The service
     # check above returns successfully as the service is running though it has
     # not even bound to a port.
-    describe command('sleep 75; curl http://localhost:8001') do
+    describe command('sleep 85; curl http://localhost:8001') do
```

Comment:
Hum...  Did a test, it needs 3 mn to start on my test machine. I got these timings (adjacent lines with similar timestamps where removed):

```sh-session
[root@centos7-64 nginx]# journalctl -u hyperglass
Jun 16 19:47:44 centos7-64.example.com systemd[1]: Started hyperglass looking glass server.
[...]
Jun 16 19:47:45 centos7-64.example.com python3[1672]: [DEBUG] 20210616 19:47:45 | hyperglass.util.frontend:313 | build_frontend → node_modules has not been initialized. Starting initialization...
Jun 16 19:49:17 centos7-64.example.com python3[1672]: [DEBUG] 20210616 19:49:17 | hyperglass.util.frontend:318 | build_frontend → Re-initialized node_modules
[...]
Jun 16 19:49:18 centos7-64.example.com python3[1672]: [DEBUG] 20210616 19:49:18 | hyperglass.util.frontend:366 | build_frontend → Created temporary UI config file: '/tmp/hyperglass_43fkmlpe.json' for build 95848d2cd30224df04d3a90706e11e
Jun 16 19:50:40 centos7-64.example.com python3[1672]: [DEBUG] 20210616 19:50:40 | hyperglass.util.frontend:385 | build_frontend → Re-initialized node_modules
[...]
Jun 16 19:50:41 centos7-64.example.com python3[1672]: [SUCCESS] 20210616 19:50:41 | hyperglass.main:138 | on_starting → Started hyperglass 1.0.0 on http://127.0.0.1:8001 with 8 workers
```

Reply:
mhm strange. maybe the provided cpu power per CI job varies highly. We had multiple runs in the past where 75s was totally fine. I will increase it slowly and see if it passes. I also had 2 CI runs today where puppet 6 jobs worked fine with the 75s and only the puppet 7 job failed.

Reply:
we always install the latest pypi release of hyperglass, the older jobs that were successul all used an older releae. v1.0.0 is quite new (10 days). maybe that increased the startup time.

Reply:
Hehe I have a crappy machine :-)

Maybe it is possible to tune the services so that `systemctl start foo` only returns when the service is up and running, not just when it has started to run?  That would be an elegant fix to this first startup time issue?

Reply:
I will check that with the developer, but as far as I know hyperglass doesn't send any signal when it's fully up and running so that will be hard to catch for systemd. but the systemd actions pass now again, so that's something :shrug: